attestation-agent/attestation-agent/Cargo.toml

```
@@ -59,13 +59,14 @@ kbs = ["kbs_protocol/background_check", "token"]
 # CoCoAS Attestation Token
 coco_as = ["reqwest", "token"]
 
-all-attesters = ["tdx-attester", "sgx-attester", "az-snp-vtpm-attester", "az-tdx-vtpm-attester", "snp-attester", "se-attester"]
+all-attesters = ["tdx-attester", "sgx-attester", "az-snp-vtpm-attester", "az-tdx-vtpm-attester", "snp-attester", "se-attester", "cca-attester"]
```

as we are now touching this line, I was wondering if it would make sense to follow 954e6d35 and move them over multiple lines:
```suggestion
all-attesters = [
    "tdx-attester",
    "sgx-attester",
    "az-snp-vtpm-attester",
    "az-tdx-vtpm-attester",
    "snp-attester",
    "se-attester",
    "cca-attester"
]
```

---

Change added to rebased commit